scipy/integrate/_quadpack_py.py

```diff
@@ -702,16 +781,27 @@ def dblquad(func, a, b, gfun, hfun, args=(), epsabs=1.49e-8, epsrel=1.49e-8):
 
     """
 
+    if epsabs != inspect.signature(dblquad).parameters.get('epsabs').default:
```

Comment:
This has a loophole: it won't raise a warning if the user explicitly passes the default tolerance. However that is `1.49e-8` and I think the chance that anyone chooses to explicitly pass the default tolerance is quite low

Reply:
You could change the default to `None` to avoid this.

Reply:
I don't feel comfortable changing the default argument if we're already going to deprecate it

Reply:
It wouldn't change the behavior though. You would check `if epsabs is None: epsabs = 1.49e-8`. But then you'd know whether that value was passed or not. It avoids the loophole you mentioned. Not important though. It's fine to leave as-is.